docs/rules/no-unused-vars.md

```diff
@@ -195,6 +195,16 @@ Examples of **correct** code for the `{ "args": "none" }` option:
 })();
 ```
 
+### ignoreRestSiblings
+
+The `ignoreRestSiblings` option is a boolean (default: `false`). With modern JavaScript it is possible to "omit" properties from an object by using a rest parameter, but by default the sibling properties are marked as "unused". With this option enabled the omitted properties are considered "used", as their existence has a side-effect. In the example, `type` is considered "used".
+
+```js
```

Comment:
To make sure the code box ends up colored green on the site to indicate "correct" code, you should put something like this before the code box:

````markdown
Examples of **correct** code for the `{ "ignoreRestSiblings": true }` option:

```js
// ...example code
```
````